init.lua

```diff
@@ -222,3 +222,21 @@ minetest.register_chatcommand("whereis", {
 				pos.x, pos.y, pos.z)
 	end
 })
+
+minetest.register_chatcommand("s", {
+    params = "<msg>",
+    description = "Send a message on the staff channel",
+    privs = {kick=true},
+    func = function(name, param)
+    local staff = {}
+    for _, player in pairs(minetest.get_connected_players()) do
+        local toname = player:get_player_name()
+        if minetest.check_player_privs(toname, {kick=true}) then
```

Comment:
It's better to also check for `ban`

```suggestion
        if minetest.check_player_privs(toname, { kick = true, ban = true }) then
```

Reply:
Same for `privs`